.azure-pipelines/cask-release.yml

```diff
@@ -0,0 +1,43 @@
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - tools/custodian-cask/*
+
+variables:
+- group: CustodianCoreCI
+
+jobs:
+- job: 'CustodianCaskPublish'
+  displayName: 'Container Cask Publish'
+  pool:
+    vmImage: 'Ubuntu-16.04'
+
+  steps:
+  - task: GoTool@0
+    inputs:
+      version: '1.12.6'
+
+  - script: make release VERSION=$(Build.BuildNumber)
+    workingDirectory: tools/cask/
+    displayName: "Build"
+
+  - task: UsePythonVersion@0
+    displayName: "Set Python Version"
+    inputs:
+      versionSpec: '3.7'
+      architecture: 'x64'
+
+  - script: pip install awscli --upgrade
```

Comment:
can ditch the upgrade this stuff has been around almost a decade. only need upgrades for newer apis.